### ISSUE TITLE
Markdown is not rendered right in gist comments

### ISSUE THREAD

**Issue by rogerbraun** [codebrawl/codebrawl#39]
Or like, not at all.

Compare http://codebrawl.com/contests/methods-taking-multiple-blocks vs. https://gist.github.com/376e6db6e7773f9a6a84

I will look into that when I have time and no one else wants to fix it.

**Comment by rogerbraun** [comment]
Had a look at the source and found out it fetches the comments via javascript directly from the gist. So I see essentially three ways to go:
- Use a JS markdown library
- Make it go through the main application, rendering the html on the server
- Just insert whatever html github is generating with jquery's load()

I would prefer the second solution. Any comments?

**Comment by jeffkreeftmeijer** [comment]
We actually use a JS markdown library, but that doesn't handle the Github specific stuff (like ```, for example). I've thought about fetching the comments on the server side, but that could push us over Github's API limit in no time...

**Comment by rogerbraun** [comment]
Yeah, I saw this after posting here... Did not think about the API limit. I'll take a look at the JS library.
